README.md

```diff
@@ -7,35 +7,23 @@
 
 ## Usage
 
-Images are served up with a URI that contains a bucket name, as well as a 
-unique identifier for the image:
-        
-        images.example.com/mybucket/5272a0e7d0d9813e21
+### Getting resized images
 
-When images are requested they are placed into an in-memory cache to make repeated
-requests for that image faster.
+Images are served up with a URI that contains a bucket name, as well as a unique identifier for the image: `http://images.example.com/mybucket/5272a0e7d0d9813e21`.
 
-You can resize an image on the fly by providing an `?s=X` parameter that specifies
-a maximum width for the image. For example, if you want to resize an image down to a 160 pixel thumbnail:
-        
-        images.example.com/mybucket/5272a0e7d0d9813e21?s=160
-
-Resized images may also be center-cropped by passing `?c=true` in the querystring:
-        
-        images.example.com/mybucket/5272a0e7d0d9813e21?s=160&c=true
+When images are requested they are placed into an in-memory cache to make repeated requests for that image faster.
 
-The thumbnail will then be cached to both `groupcache` and S3. If the image leaves
-the in-memory cache it will not need to be resized again.
+You can resize an image on the fly by providing an `?s=X` parameter that specifies
+a maximum width for the image. For example, if you want to resize an image down to a 160 pixel thumbnail: `http://images.example.com/mybucket/5272a0e7d0d9813e21?s=160`. Resized images may also be center-cropped by passing `?c=true` in the querystring: `images.example.com/mybucket/5272a0e7d0d9813e21?s=160&c=true`. The thumbnail will then be cached to both `groupcache` and S3. If the image leaves the in-memory cache it will not need to be resized again.
 
```

Comment:
IMO, the above paragraph looked a little better with the URLs on their own lines. In the new one, the urls are big code blocks that cause line-length weirdness anyway, so breaking them out ends up clearer and less funky.